cli/src/test/cli.spec.ts

```diff
@@ -0,0 +1,40 @@
+import test from 'ava';
+import { Shell, success } from 'specshell';
+import * as R from 'ramda';
+
+const process = R.evolve({ out: (x: Buffer) => x.toString(), err: (x: Buffer) => x.toString() });
+
+test('cli with no args shows VERSION, USAGE, COMMANDS', async (t) => {
```

Comment:
That's not what this tests

Reply:
Sorry, pushed the pre-rebase version.

Reply:
Still wrong..
```suggestion
test('cli with no args exits successfully', async (t) => {
```

Reply:
Yeah, that test dropped an early assertion.  Restored that one.